src/check.js

```diff
@@ -0,0 +1,16 @@
+function displayBox() {
+  const labelList = document.querySelectorAll('.list');
+  const checkBoxes = document.querySelectorAll('.check-list');
+  for (let i = 0; i < checkBoxes.length; i += 1) {
+    checkBoxes[i].addEventListener('change', () => {
+      if (checkBoxes[i].checked) {
+        labelList[i].style.textDecoration = 'line-through';
+      } else {
+        labelList[i].style.textDecoration = 'none';
+      }
+    });
+  }
+}
```

Comment:
- [ ] It would be better if you add a method that checks that changes the status `completed` to true or false. So when you check the box the task status change accordingly. 

![tempsnip](https://user-images.githubusercontent.com/64173189/144303309-68ca7035-35b3-4afa-9d56-1e743ff77037.png)


- [ ] Please make sure that the status of the `completed` is saved in local storage so when you refresh the task keep the correct value for the tasks.